live_python_qa/utils.py

```diff
@@ -41,12 +41,12 @@ def estimateggdparam(vec):
     r_gam = (gamma(1.0/gam)*gamma(3.0/gam))/((gamma(2.0/gam))**2)
     # print(np.mean(vec))
     sigma_sq = np.mean(vec**2)  #-(np.mean(vec))**2
-    sigma = np.sqrt(sigma_sq)
+    #sigma = np.sqrt(sigma_sq)
     E = np.mean(np.abs(vec))
     rho = sigma_sq / (E**2 + 1e-6)
     array_position = (np.abs(rho - r_gam)).argmin()
     alphaparam = gam[array_position]
-    return alphaparam, sigma
+    return alphaparam, sigma_sq
```

Comment:
Let's make sure that every function that uses `estimateggdparam` is updated to reflect this change.

Reply:
sigma^2.
You can  check here: https://github.com/utlive/BRISQUE/blob/bd088c3f6b745629caa02f663e4665cb51591ac0/MATLAB/brisque_feature.m#L21

In the Matlab code, estimateggdparam returns sigma but in the brisque_feature.m it is squared and added to the feature vector. Even the paper mentions using sigma^2.

I will check for compatibility with NIQE by the weekend and update the code.

Reply:
Thanks for catching that!